packages/agreement-email-sender/src/config/config.ts

```diff
@@ -23,31 +24,12 @@ export const EmailSenderConfig = z
   }));
 export type EmailSenderConfig = z.infer<typeof EmailSenderConfig>;
 
-export const PecEmailManagerConfig = z
-  .object({
-    PEC_SMTP_ADDRESS: z.string(),
-    PEC_SMTP_PORT: z.coerce.number(),
-    PEC_SMTP_SECURE: z
-      .enum(["true", "false"])
-      .transform((value) => value === "true"),
-    PEC_SMTP_USERNAME: z.string(),
-    PEC_SMTP_PASSWORD: z.string(),
-  })
-  .transform((c) => ({
-    pecSmtpAddress: c.PEC_SMTP_ADDRESS,
-    pecSmtpPort: c.PEC_SMTP_PORT,
-    pecSmtpSecure: c.PEC_SMTP_SECURE,
-    pecSmtpUsername: c.PEC_SMTP_USERNAME,
-    pecSmtpPassword: c.PEC_SMTP_PASSWORD,
-  }));
-export type PecEmailManagerConfig = z.infer<typeof PecEmailManagerConfig>;
-
```

Comment:
If these `PEC_*` vars are not needed anymore, also update the `.env` file in `agreement-email-sender` to remove them